kotlin/src/main/kotlin/org/jdbi/v3/core/kotlin/Exceptions.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core.kotlin
+
+import org.jdbi.v3.core.JdbiException
+
+
+class KotlinMemberAccessException : JdbiException {
+
+    constructor(string: String, throwable: Throwable) : super(string, throwable) {}
+
+    constructor(cause: Throwable) : super(cause) {}
+
+    constructor(message: String) : super(message) {}
+
+    companion object {
+        private val serialVersionUID = 1L
+    }
+}
+
+class NoSuchColumnMapperException : JdbiException {
```

Comment:
There's a `NoSuchMapperException` (for both row and column) in core, if you want to use that.